data/actionsmap/yunohost.yml

```diff
@@ -1351,6 +1351,16 @@ tools:
                     help: Show private data (domain, IP)
                     action: store_true
 
+        ### tools_portavailable()
+        portavailable:
```

Comment:
The most widely used convention seems to imply "portavailable". (Even though I'm the one who introduced cert-install, cert-status and cert-renew :confused: )